src/poh_service.rs

```diff
@@ -140,7 +165,8 @@ mod tests {
         };
 
         const HASHES_PER_TICK: u64 = 2;
-        let poh_service = PohService::new(poh_recorder, Config::Tick(HASHES_PER_TICK as usize));
+        let (tx, _) = channel();
```

Comment:
`tx` is widely used in the codebase to mean "transaction". Let's not use tx/rx for transmitter/receiver as well.

Reply:
sender for senders..., receivers for...

Reply:
Gotcha. I really need to get this working before I clean up though :D that's why still left it as wip.